pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -611,7 +611,7 @@ func TestReconcile(t *testing.T) {
 				tb.PodContainer("step-git-source-git-resource-mz4c7", "override-with-git:latest",
 					tb.Command(entrypointLocation),
 					tb.Args("-wait_file", "/tekton/tools/0", "-post_file", "/tekton/tools/1", "-entrypoint", "/ko-app/git-init", "--",
-						"-url", "https://foo.git", "-revision", "master", "-path", "/workspace/workspace"),
+						"-url", "https://foo.git", "-revision", "master", "-path", "/workspace/workspace", "-sslVerify=true"),
```

Comment:
since SSLVerify defaults to true, one option is to only provide it when it's false, which would simplify the arguments to this binary and in a lot of test cases :thinking: